lib/ruby-debug-ide/attach/util.rb

```diff
@@ -1,5 +1,16 @@
 require 'ruby-debug-ide/attach/lldb'
 require 'ruby-debug-ide/attach/gdb'
+require 'socket'
+
+def get_child_pids(pid, pids = Array.new)
```

Comment:
Using mutable container is not the best solution unless you're working with tens of thousands of elements